src/Util/ReversedFileReader.php

```diff
@@ -158,7 +158,7 @@ public function current(): string
 	}
 
 	/**
-	 * Checks if current iterator value is valid, that is, we readed all lines in files
+	 * Checks if current iterator value is valid, that is, we read all lines in files
```

Comment:
I'm not entirely certain about this change

Reply:
Unfortunately, while the fix is correct, it doesn't help much with this counter-intuitive class.